src/styles.css

```diff
@@ -0,0 +1,123 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgba(183, 164, 164, 0.986);
+}
+
+.list-container {
+  background-color: #fff;
+  width: 45%;
+  height: auto;
+  position: absolute;
+  top: 25%;
+  left: 50%;
+  transform: translate(-50%, -25%);
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+.bi-journal-text {
+  font-size: 5rem;
+  color: rgb(29, 125, 185);
+  margin: 15px auto;
+}
```

Comment:
- Kindly, stick to the given design to fit the [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md) and have better UI 👍 
#### your Desgn 👇
![Screenshot 2023-07-12 164751](https://github.com/riskmaris/ToDo-List-App/assets/52010789/81ce7e8f-bf23-498a-936b-d6cb6ca75f6e)
#### Given Design 👇
![Screenshot 2023-07-12 164816](https://github.com/riskmaris/ToDo-List-App/assets/52010789/31e79b7a-ad21-4168-9d6e-07366e5db21e)